### PR TITLE
Code reviewer role

### PR DESCRIPTION
**Pull request by victormc13**
### Latest Features

-Add Html file and CSS to the Survey form

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,57 +1,56 @@
...
+  font-size: 2.5rem;
+  font-weight: 400;
+  line-height: 1.5;
 }
 
 #title,
 #description {
-    text-align: center;
+  text-align: center;
 }
```
Comment (victormc13) [inline css/style.css:32]: -If you have to assignment style CSS for the same tag you can use a class or id and write the code in the same part of the selector for the same class or id, for clean code.

File: css/style.css [hunk 2]
```diff
@@ -1,57 +1,56 @@
...
+
 h1,
-  p {
-    margin-top: 0;
-    margin-bottom: 0.5rem;
-  }
+p {
+  margin-top: 0;
+  margin-bottom: 0.5rem;
+}
```
Comment (victormc13) [inline css/style.css:47]: There may be times when two or more declarations are applied to the same element. It is also possible that there may be a conflict between them.
 
Remember that CSS is declared last in the CSS document and therefore, overrides the previous declarations 🤔

**Review by victormc13 — COMMENTED** [review]
Hi @victormc13 👋🏽 
Good job! There are just a few more issues to work on before moving on, but you are almost there! 🤏🏽
**Highlights**
- Descriptive Pull Request message✔️
- Good use of GitHub flow✔️
- Meaningful commit messages✔️
- No linter errors✔️
- Good design✔️
###  Changes Required ♻️
Please check out my comments below and make all required changes.
Submit it for a re-review when you are done, and if you have any concerns or questions, let me know.

Thanks. 😁

File: css/style.css (hunk 1 above)
Comment (Shedrack-Sunday) [reply css/style.css:32]: - [ ] You did well suggesting a change here, however, always add a "why" for that change suggested. this gives clarity to the project owner as to why such change is needed for example from your review, I can add:

`.......make it easier to make changes to specific elements without affecting others. Additionally, using classes and ids can make your CSS more organized and easier to read and maintain.`

**Review by victormc13 — COMMENTED** [review]
Hi! @victormc13 👋🏽

Good job with this so far!
There are just a few more issues to work on before moving on, but you are almost there! 🤏🏽

**Highlights**
-Descriptive Pull Request message ✔️
-Good use of GitHub flow ✔️
-Meaningful commit messages ✔️
-No linter error ✔️

**Changes Required ♻️** 

Please check my comments below and make all required changes.
Submit it for a re-review when you complete all required changes, and if you have any concerns or questions, just let me know.

Thanks. 😁

**Review by amejid — APPROVED** [review]
Hi @victormc13, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good use of Github features :heavy_check_mark: 
- Comments are polite and provide a reason :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**